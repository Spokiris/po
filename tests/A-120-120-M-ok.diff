*** tests/A-120-120-M-ok.out	2023-10-19 17:17:27.000000000 +0100
--- tests/A-120-120-M-ok.outhyp	2023-10-27 19:48:58.675774016 +0100
***************
*** 1,17 ****
  Menu Principal
  1 - Novo
  2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
  1 - Novo
  2 - Abrir
- 3 - Guardar
- 4 - Menu de Edição
- 5 - Menu de Consultas
  0 - Sair
  Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
  1 - Novo
  2 - Abrir
--- 1,13 ----
  Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
! Escolha uma opção: Opção inválida!
! Menu Principal
  1 - Novo
  2 - Abrir
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
  1 - Novo
  2 - Abrir
***************
*** 54,66 ****
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Guardar antes de fechar? Ficheiro a abrir: Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
  Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 51,65 ----
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir ficheiro: n (No such file or directory)
! Menu Principal
  1 - Novo
  2 - Abrir
  3 - Guardar
  4 - Menu de Edição
  5 - Menu de Consultas
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 70,76 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|'789=COALESCE(1;1:1;3)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 69,76 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|5
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
***************
*** 80,86 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;4|1075=PRODUCT(2;1:2;2)
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 80,87 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 
! 
  Menu de Edição
  1 - Visualizar
  2 - Inserir
