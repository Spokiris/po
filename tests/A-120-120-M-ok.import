linhas=41
colunas=32
1;3|'123
1;2|'456
1;1|'789
2;1|25
2;2|43
2;3|=COALESCE(1;1:1;3)
2;4|=PRODUCT(2;1:2;2)
