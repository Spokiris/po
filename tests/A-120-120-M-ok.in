3
saved30
1
12
12
4
2
2;3
5
1
2;3
0
2
n
saved30
4
1
2;3
1
2;4
0
0
