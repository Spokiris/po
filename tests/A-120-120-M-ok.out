Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Especifique o número de linhas da folha: Especifique o número de colunas da folha: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|5
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Guardar antes de fechar? Ficheiro a abrir: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;3|'789=COALESCE(1;1:1;3)
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 2;4|1075=PRODUCT(2;1:2;2)
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 